UpdateDependencies.ps1

```diff
@@ -32,8 +33,14 @@ function UpdateValidDependencyVersionsFile
 
     $DirPropsPath = "$PSScriptRoot\dir.props"
     
-    $DirPropsContent = Get-Content $DirPropsPath | % { 
-        $_ -replace "<CoreFxExpectedPrerelease>.*</CoreFxExpectedPrerelease>","<CoreFxExpectedPrerelease>$CoreFxLatestVersion</CoreFxExpectedPrerelease>"
+    $DirPropsContent = Get-Content $DirPropsPath | % {
+        $line = $_
+        $DirPropsVersionElements | % {
+            $line = $line -replace `
+                "<$_>.*</$_>", `
+                "<$_>$CoreFxLatestVersion</$_>"
```

Comment:
Good idea, changed. This led me to the commit title (and PR title) where I added the name of the element being updated to hopefully make it a bit clearer to reviewers. The names might be a little long, though...